rest/taskana-rest-spring/src/test/java/pro/taskana/common/rest/GeneralExceptionHandlingTest.java

```diff
@@ -1,48 +1,40 @@
 package pro.taskana.common.rest;
 
 import static org.assertj.core.api.Assertions.assertThatThrownBy;
+import static pro.taskana.common.test.rest.RestHelper.TEMPLATE;
 
 import org.assertj.core.api.ThrowableAssert.ThrowingCallable;
-import org.junit.jupiter.api.BeforeAll;
 import org.junit.jupiter.api.Test;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.core.ParameterizedTypeReference;
 import org.springframework.http.HttpMethod;
 import org.springframework.web.client.HttpClientErrorException;
-import org.springframework.web.client.RestTemplate;
 
-import pro.taskana.classification.rest.models.ClassificationSummaryRepresentationModel;
-import pro.taskana.common.rest.models.TaskanaPagedModel;
+import pro.taskana.classification.rest.models.ClassificationSummaryPagedRepresentationModel;
 import pro.taskana.common.test.rest.RestHelper;
 import pro.taskana.common.test.rest.TaskanaSpringBootTest;
 
 /** Test general Exception Handling. */
 @TaskanaSpringBootTest
 class GeneralExceptionHandlingTest {
 
-  private static final ParameterizedTypeReference<
-          TaskanaPagedModel<ClassificationSummaryRepresentationModel>>
-      CLASSIFICATION_SUMMARY_PAGE_MODEL_TYPE =
-          new ParameterizedTypeReference<
-              TaskanaPagedModel<ClassificationSummaryRepresentationModel>>() {};
-  private static RestTemplate template;
-  @Autowired RestHelper restHelper;
+  private final RestHelper restHelper;
 
-  @BeforeAll
-  static void init() {
-    template = RestHelper.TEMPLATE;
+  @Autowired
+  GeneralExceptionHandlingTest(RestHelper restHelper) {
+    this.restHelper = restHelper;
   }
 
   @Test
   void testDeleteNonExisitingClassificationExceptionIsLogged() {
```

Comment:
test naming again.....again, you can ignore this if you don't feel like changing it